tasks/12.txt

```diff
@@ -3,3 +3,10 @@ Write a SQL query to get Product list of twenty most expensive products:
 | ProductName | UnitPrice |
 
 Order products by price.
+
+db.products.aggregate(
+    [
+    {$project: {ProductName: 1, UnitPrice: 1}},
+    {$sort: {UnitPrice: -1}},
+    {$limit : 20}
+    ]);
```

Comment:
Sort and limit should be used as soon as logic allow that

Reply:
In your case you lost all indexes(if you  have or will have) after firs $project step